### PR TITLE
add my task 4

### PR DESCRIPTION
**Pull request by VeronikaZhagun**

### CODE REVIEW

**Comment by VeronikaZhagun** [comment]
I'm sorry, I did something strange here. Hope you figure it out

File: Tasks/Zhagun/Task4/Hometask4.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+def check_str(string):
+    counter={'upper_case':0,'lower_case':0}
+    for i in string:
+        if i.isupper():
+            counter['upper_case']+=1
+        elif i.islower():
+            counter['lower_case']+=1
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Zhagun/Task4/Hometask4.py:7]: good one

File: Tasks/Zhagun/Task4/Hometask4.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        else:
+            pass
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Zhagun/Task4/Hometask4.py:9]: the redundant part)

File: Tasks/Zhagun/Task4/Hometask4.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        if l[i + 1] - l[i] == 1:
+            Flag = True
+        else:
+            if Flag:
+                list += f'-{l[i]}, {l[i + 1]}'
+            else:
+                list += f', {l[i + 1]}'
+            Flag = False
+    if Flag:
+        list += f'-{l[-1]}'
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Zhagun/Task4/Hometask4.py:32]: an interesting approach to the result handling

**Review by MikitaTsiarentsyeu — APPROVED** [review]
good job, the score is 100

**Pull request merged**